di-ipv-cimit-stub/deploy/template.yaml

```diff
@@ -373,6 +380,68 @@ Resources:
               ArnLike:
                 "kms:EncryptionContext:aws:logs:arn": !Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:*"
 
+  # ssl cert
+  CimitStubSSLCert:
+    Type: AWS::CertificateManager::Certificate
+    Properties:
+      DomainName: !If
+        - IsCoreDev
+        - !Sub "cimit-${Environment}.core.dev.stubs.account.gov.uk"
+        - !If [ IsNonProd, !Sub "cimit.${Environment}.stubs.account.gov.uk", cimit.stubs.account.gov.uk ]
+      DomainValidationOptions:
+        - DomainName: !If
+            - IsCoreDev
+            - !Sub "cimit-${Environment}.core.dev.stubs.account.gov.uk"
+            - !If [ IsNonProd, !Sub "cimit.${Environment}.stubs.account.gov.uk", cimit.stubs.account.gov.uk ]
+          HostedZoneId: !If
+            - IsCoreDev
+            - !ImportValue DevPublicHostedZoneId
+            - !If [ IsNonProd, !ImportValue BuildPublicHostedZoneId, !ImportValue RootPublicHostedZoneId ]
+      ValidationMethod: DNS
+
+  # api domain entries / mapping
+  CimitStubRestApiDomain:
```

Comment:
We are not using apigateway v2 - this uses a rest API gateway so you'll need something likethis instead:
`  # api domain entries / mapping
  RestApiGatewayDomain:
    Type: AWS::ApiGateway::DomainName
    Condition: IsNewDevelopment
    Properties:
      DomainName: !Sub "rest.${Environment}.dev.identity.account.gov.uk"
      RegionalCertificateArn: !Ref RestSSLCert
      EndpointConfiguration:
        Types:
          - REGIONAL

  RestApiBasePathMapping:
    Type: AWS::ApiGateway::BasePathMapping
    Condition: IsNewDevelopment
    Properties:
      DomainName: !Ref RestApiGatewayDomain
      RestApiId: !Ref RestApiGateway
      Stage: !Ref ApiGatewayStage`

Reply:
Thank you for the feedback. I have made the changes and retest again with Postman. It's working with version changes.